source/collaborate/channel-header-purpose.rst

```diff
@@ -41,9 +41,34 @@ A channel purpose can be up to 250 characters in length, and is often used to he
 .. tab:: Mobile
 
   1. Tap the channel you want to edit.
-  2. Long-press on the channel purpose.
-  3. Tap **Copy purpose text**.
-  4. Tap the purpose field to paste the copied text.
+
```

Comment:
@Rutam21 - The new code you've added is intended to the right by a single character, and this is enough to cause display issues (see the generated preview for details).

Removing the extra space on each code line should resolve the display issues on this page (both sections).

Reply:
I have already updated it. Please re-check. Thanks.

Reply:
Looks like you were applying these updates as I was commenting on them! Great minds think alike! :)

Reply:
Yeah, indeed. I was also checking the preview after it was generated.

Reply:
Latest preview continues to show the images as clickable links. 

What's baffling is that mobile #5 in both sections displays as intended. I have yet to confirm why it's working and the rest aren't. I may need to pull down this fork locally and investigate further.